src/main/java/org/folio/inventory/consortium/entities/Status.java

```diff
@@ -0,0 +1,7 @@
+package org.folio.inventory.consortium.entities;
+
+public enum Status {
```

Comment:
If this is referring to status of instance sharing, i think this class should be called SharingStatus. The name is very generic right now.

Reply:
Thanks, done